test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <Optimize>true</Optimize>
+    <Optimize>false</Optimize>

```

Comment:
was this meant for checkin?

Reply:
@ElizabethOkerio I'm considering checking this in. When `Optimize` is set to `true` and you place a breakpoint on any line of code within the project, the breakpoint won't be hit! I don't know if this setting is meant to speed up the tests on the build environment but it cost me quite a bit of time to figure out why my breakpoints weren't being hit. I don't think it's worth leaving this setting with a value of `true` since this is a test project and once in a while one wants to step through the code in the test project

Reply:
@habbes In this case it's the test project that is optimized but the referenced project is not. Unless I didn't understand your concern